### DISCUSSION TITLE
A "Scheduled messages" folder

### DISCUSSION THREAD

**Discussion by andreamoretti** [Extended-Thunder/send-later#473] [Ideas]
Ciao,
why not creating a folder only for scheduled messages? Or, at least, give the chance to choose it.
I think that could be better for the user to don't mix draft and scheduled messages.

Ciao e grazie

Andrea Moretti

**Comment by jikamens** [comment]
Tracking this in #635.

**Comment by jikamens** [comment]
This issue should be fixed in release 10.2.0 of Send Later. To check what version of Send Later you have:

* Open the Add-ons page in Thunderbird ("Add-ons and Themes" from the corner "hamburger" menu).
* Click on Extensions on the left if you're not already on the left if you're not already there.
* Find Send Later in the list of extensions and click on it to open its details page.
* The version number should be displayed there next to
"Version".

This release is not yet available for download from addons.thunderbird.net, but you can download and install it from its [GitHub release page](https://github.com/Extended-Thunder/send-later/releases/tag/v10.2.0) as follows:

* Download [this file](https://github.com/Extended-Thunder/send-later/releases/download/v10.2.0/send_later.xpi) to your computer. (If you are using Firefox, make sure to right click on the link and select
"Save Link As..." rather than just clicking on it, because otherwise Firefox will try to install it as a Firefox add-on.)
* Open the Add-ons page in Thunderbird ("Add-ons and Themes" from the corner "hamburger" menu).
* Click the gear icon at the top and select "Install Add-on From File...".
* Browse to and select the downloaded file.
* Click through the installation pop-ups.
* You can delete the downloaded file once you've installed it into Thunderbird.

Note that this is a prerelease. While we make every effort to ensure that prereleases are stable, they are a bit more likely to have bugs, so proceed with caution.

If you prefer to wait, the release will eventually be available on addons.thunderbird.net, and unless you've turned off automatic add-on updates, Thunderbird will eventually update to the new version automatically. To update immediately once it's available:

* Go to [addons.thunderbird.net](https://addons.thunderbird.net/thunderbird/addon/send-later-3/versions/) and confirm that release 10.2.0 is available for download there.
* Open the Add-ons page in Thunderbird ("Add-ons and Themes" from the corner "hamburger" menu).
* Click on Send later.
* Click the gear icon and select "Check for Updates".
* The add-on should update and then you should see the message, "Your add-ons have been updated," next to the gear icon.

Please let me know if you still see this issue in this release.
